samples/peripheral/radio_test/src/radio_test.c

```diff
@@ -91,6 +91,92 @@ static uint16_t channel_to_frequency(nrf_radio_mode_t mode, uint8_t channel)
 #endif /* CONFIG_HAS_HW_NRF_RADIO_IEEE802154 */
 }
 
+static nrf_radio_txpower_t dbm_to_nrf_radio_txpower(int8_t tx_power)
+{
+	switch (tx_power) {
+	case -40:
+		return RADIO_TXPOWER_TXPOWER_Neg40dBm;
+
+	case -30:
+		return RADIO_TXPOWER_TXPOWER_Neg30dBm;
+
+	case -20:
+		return RADIO_TXPOWER_TXPOWER_Neg20dBm;
+
+	case -16:
+		return RADIO_TXPOWER_TXPOWER_Neg16dBm;
+
+	case -12:
+		return RADIO_TXPOWER_TXPOWER_Neg12dBm;
+
+	case -8:
+		return RADIO_TXPOWER_TXPOWER_Neg8dBm;
+
+#if defined(RADIO_TXPOWER_TXPOWER_Neg7dBm)
+	case -7:
+		return RADIO_TXPOWER_TXPOWER_Neg7dBm;
+#endif /* defined(RADIO_TXPOWER_TXPOWER_Neg7dBm) */
+#if defined(RADIO_TXPOWER_TXPOWER_Neg6dBm)
+	case -6:
+		return RADIO_TXPOWER_TXPOWER_Neg6dBm;
+#endif /* defined(RADIO_TXPOWER_TXPOWER_Neg6dBm) */
+#if defined(RADIO_TXPOWER_TXPOWER_Neg5dBm)
```

Comment:
```suggestion
#endif /* defined(RADIO_TXPOWER_TXPOWER_Neg7dBm) */

#if defined(RADIO_TXPOWER_TXPOWER_Neg6dBm)
	case -6:
		return RADIO_TXPOWER_TXPOWER_Neg6dBm;
#endif /* defined(RADIO_TXPOWER_TXPOWER_Neg6dBm) */

#if defined(RADIO_TXPOWER_TXPOWER_Neg5dBm)
```

Similarly in other places.

Reply:
Corrected